README.md

```diff
@@ -23,7 +23,7 @@
 
 - Define and clean gene models
 
-![isoseq_pipeline_graph](docs/images/Isoseq_pipeline_V2.1-metro_clean.png)
+![isoseq_pipeline_graph](docs/images/Isoseq_pipeline_V2.3-metro_clean.png)
```

Comment:
How is the pipeline version 1.1.3 but the metro map version 2.3?

Reply:
Those version numbers are not correlated. I just gave a version number to the image while I was drawing it.

Reply:
It is a bit confusing.
I'd rather just have `docs/images/Isoseq_pipeline_metro.png`

Reply:
Isn't git doing the versioning for us, and I'm assuming there's only one version for each release?

Reply:
You're right, it was for personal purpose at beginning. 
I renamed the file and updated the README.md :)